content/how-to/market-data/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Market Data Examples
+weight: 10
+---
+
+Alpaca provides market data from various sources. To learn more about data options and which one is right for you, please see [this page](https://docs.alpaca.markets/web-api/market-data/).
+
+---
+
+## Get Historical Price and Volume Data
+
+By making a GET request to our [`/v1/bars`](https://docs.alpaca.markets/web-api/market-data/bars/) endpoint, you can see what a stock price was at a particular time.
+
+{{< code-example exampleId="getPrices" pathURL="/how-to/market-data/examples">}}
+
+## Listen for Current Market Data with Websockets
+
+You can use Websockets to receive real-time updates about market prices.
+
+{{< code-example exampleId="streaming" pathURL="/how-to/market-data/examples">}}
```

Comment:
Please hold on with the bars streaming in data API. It's not fully released yet.